.github/ISSUE_TEMPLATE/improvement-request.md

```diff
@@ -3,27 +3,24 @@ name: Improvement request
 about: Use this option when you have defined an existing area that can be improved.
   Please attach relevant research documentation so that your issue can be handled
   at the UX Management Group.
-title: ''
+title: Improvement - "Add title"
 labels: Improvement
 assignees: ''
 
 ---
 
-**Describe the interaction pattern**<br/>
-If this improvement is connected to an existing interaction pattern, component or element, please describe it here (e.g Date Picker, Button Drop Down, etc.)
+**Describe the improvement**
```

Comment:
Because it is an improvement of something, add another section for "Describe current situation or problem"
And then we can describe the improvement we want in the next section.

Reply:
Updated